src/EditorFeatures/Test2/ReferenceHighlighting/CSharpReferenceHighlightingTests.vb

```diff
@@ -508,7 +508,28 @@ class C
     {
         var a = "Hello";
         var $${|Definition:b|} = "World";
-        var c = $"{a}, {[|b|]}!";
+        var c = $"{a}, { {|Reference:b|} }!";
+    }
+}
+                    </Document>
+                </Project>
+            </Workspace>
+
+            VerifyHighlights(input)
+        End Sub
+
+        <Fact, Trait(Traits.Feature, Traits.Features.ReferenceHighlighting)>
+        Public Sub TestWrittenReference()
```

Comment:
Probably want more tests for various kinds of members, qualified and unqualified, etc.